cranelift/codegen/src/machinst/abi_impl.rs

```diff
@@ -196,6 +196,19 @@ pub enum ABIArg {
         /// Purpose of this arg.
         purpose: ir::ArgumentPurpose,
     },
+    /// Implicit argument. Similar to a StructArg, except that we have the
+    /// target type, not a pointer type, at the CLIF-level. This argument is
+    /// still being passed via reference implicitly.
+    ImplicitArg {
```

Comment:
Can we call this an `ImplicitPtrArg` or similar? Just `ImplicitArg` seems slightly ambiguous to me; like the arg itself is implicitly generated at callsites or something like that.

Reply:
Done